specification/security/resource-manager/readme.md

```diff
@@ -78,22 +78,31 @@ These are the global settings for the Security API.
 title: SecurityCenter
 description: API spec for Microsoft.Security (Azure Security Center) resource provider
 openapi-type: arm
-tag: package-composite-v3
+tag: package-preview-2023-08
```

Comment:
this is the default - don't change this.
you need to replace the pricing.json api-version used in the package-composite-v3